features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -606,6 +606,14 @@
             -->
             <LogoutTokenExpiration>120</LogoutTokenExpiration>
 
+            <!-- Logout request sender configurations. Time configs are in milliseconds. -->
+            <LogoutRequestSender>
+                <PoolSize>2</PoolSize>
```

Comment:
Is 2 enough?

Reply:
Didn't try to change the current default. But we can make it 10 or 20

Reply:
Addressed via 580eac29fd71e8d8e86cd1b9762cd1770527030c